internal/samsahai/controller.go

```diff
@@ -438,6 +438,20 @@ func (c *controller) createNamespaceByTeam(teamComp *s2hv1beta1.Team, teamNsOpt
 		if err := c.updateTeamNamespacesStatus(teamComp, teamNsOpt); err != nil {
 			return errors.Wrap(err, "cannot update team conditions when create namespace success")
 		}
+
+		if nsConditionType == s2hv1beta1.TeamNamespaceStagingCreated {
+			if err := c.notifyComponentChanged(teamComp.Name); err != nil {
```

Comment:
if we cannot notify the component changed, should we retry?
there might be something wrong with their values file